Mage/src/main/java/mage/cards/ExpansionSet.java

```diff
@@ -675,4 +675,13 @@ public int getNumBoosterDoubleFaced() {
         return numBoosterDoubleFaced;
     }
 
+    protected static void addCardInfoToList(List<CardInfo> boosterList, String name, String expansion, String cardNumber) {
+        CardInfo cardInfo = CardRepository.instance.findCardWithPreferredSetAndNumber(name, expansion, cardNumber);
+        if (cardInfo != null) {
```

Comment:
not null only, but set code and number too

Reply:
Good call - that caught an error, thanks